web/html/src/components/input/Form.js

```diff
@@ -16,14 +16,13 @@ type Props = {
 };
 
 type State = {
-  isValid: boolean,
+  fieldsStatus: Object,
 };
 
 type FormContextType = {
   model: Object,
   setModelValue: Function,
-  validateForm: Function,
-  isValid: boolean,
+  notifyFieldValidation: (string, boolean) => void,
```

Comment:
IMHO, it's more convenient to name this `onFieldValidation`

Reply:
:heavy_check_mark: